sys/include/net/netopt.h

```diff
@@ -37,6 +37,14 @@ extern "C" {
  */
 #define NETOPT_MAX_PACKET_SIZE NETOPT_MAX_PDU_SIZE
 
+/**
+ * @brief       A deprecated alias for @ref NETOPT_LINK
+ *
+ * @deprecated  Please use @ref NETOPT_LINK instead of
+ *              `NETOPT_LINK_CONNECTED`
```

Comment:
Please specify the release this option is slated for removal as well. `NETOPT_MAX_PACKET_SIZE` isn't doing this either, but we [just decided](https://github.com/RIOT-OS/RIOT/pull/12501#issuecomment-547827020) it is time to remove it ;-).

Reply:
ok!

Reply:
addressed in https://github.com/RIOT-OS/RIOT/pull/12603/commits/13a074b50d8dd4e895906c49727113f45f903d30